docs/tutorials/tutorials/web3auth-login.mdx

```diff
@@ -0,0 +1,320 @@
+---
+title: FuseBox Social Login using Web3Auth
+sidebar_position: 7
+---
+
+In this tutorial, we'll guide you through integrating web3 authentication into a React application using the `web3auth` library and interacting with the Fuse blockchain using the `Fusebox Web SDK.` By the end of this tutorial, you'll have a functional web application allowing users to log in with their wallets and perform basic blockchain interactions using FuseBox.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- Node.js installed on your machine
+- An account on [web3auth.io](https://web3auth.io/) to obtain your `clientId`
+- An API Key from the Fuse [Console](https://console.fuse.io)
+
+## Step 1: Setup a React App
+
+Create a new React app using the following commands:
+
+```bash
+npx create-react-app web3auth-fusebox-tutorial
+cd web3auth-fusebox-tutorial
+```
+
+## Step 2: Install Dependencies
+
+Install the necessary dependencies:
+
+```bash
+npm install @web3auth/modal @web3auth/base @fuseio/fusebox-web-sdk web3 ethers
+```
+
+## Step 3: Set Up Components
+
+Replace the contents of `src/App.js` with the following code:
+
+```jsx
+// Import necessary libraries and components
+import { useEffect, useState } from "react";
+import { Web3Auth } from "@web3auth/modal";
+import { CHAIN_NAMESPACES, IProvider } from "@web3auth/base";
+import { FuseSDK } from "@fuseio/fusebox-web-sdk";
+import { ethers } from "ethers";
+import Web3 from "web3";
+
+import "./App.css";
+
+// Initialize Web3Auth with your clientId and chain information
+const clientId = "YOUR_CLIENT_ID"; // Replace with your actual clientId
+const chainConfig = {
+  chainNamespace: CHAIN_NAMESPACES.EIP155,
+  chainId: "0x7A", // Please use 0x1 for Mainnet
+  rpcTarget: "https://rpc.fuse.io",
+  displayName: "Fuse Mainnet",
+  blockExplorer: "https://explorer.fuse.io/",
+  ticker: "FUSE",
+  tickerName: "Fuse",
+};
+
+const web3auth = new Web3Auth({
+  clientId,
+  chainConfig,
+  web3AuthNetwork: "sapphire_mainnet",
```

Comment:
@emmaodia Shouldn't it be `fuse`?

Reply:
It is actually an optional param from initializing web3auth network. It is their default network for managing session IDs if a user wants to implement one. I'll remove it as it is not critical to this guide. https://web3auth.io/docs/sdk/pnp/web/no-modal/initialize#web3authnomodaloptions